charts/templates/statefulset.yaml

```diff
@@ -21,6 +21,45 @@ spec:
       affinity: {{- toYaml . | nindent 8}}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
+      {{- if ne .Values.bootstrapHostName "" }}
+      initContainers: 
+      - name: check-group-init-container
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
We should use a lightweight image for init-container, something like curlimages/curl. Make this configurable in the values file.

Reply:
changed image for init container to curlimages/curl